releases/v1.6.27.toml

```diff
@@ -0,0 +1,40 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+# project_name is used to refer to the project in the notes
+project_name = "containerd"
+
+# github_repo is the github project, only github is currently supported
+github_repo = "containerd/containerd"
+
+# match_deps is a pattern to determine which dependencies should be included
+# as part of this release. The changelog will also include changes for these
+# dependencies based on the change in the dependency's version.
+match_deps = "^github.com/(containerd/[a-zA-Z0-9-]+)$"
+
+# previous release of this project for determining changes
+previous = "v1.6.26"
+
+# pre_release is whether to include a disclaimer about being a pre-release
+pre_release = false
+
+# preface is the description of the release which precedes the author list
+# and changelog. This description could include highlights as well as any
+# description of changes. Use markdown formatting.
+preface = """\
+The twenty-seventh patch release for containerd 1.6 contains various fixes and updates.
+
+### Notable Updates
+
+* **Improve `/etc/group` handling when appending groups** ([#9543](https://github.com/containerd/containerd/pull/9543))
+* **Update runc binary to v1.1.11** ([#9597](https://github.com/containerd/containerd/pull/9597))
+* **Remove runc import** ([#9606](https://github.com/containerd/containerd/pull/9606))
+* **Update shim pidfile permissions to 0644** ([#9613](https://github.com/containerd/containerd/pull/9613))
+
```

Comment:
Can the go version change also be added here? #9625

Reply:
I don't usually include unless there is a specific security issue or bug included with it. Was there something specific? We have discussed about just having the Go version be something that automatically gets included in the notes

Reply:
Updated

Reply:
There was no security fix, just fixes to crypto package and runtime.